### ISSUE TITLE
Improve on WCAG "Robust" category - a11y audit

### ISSUE THREAD

**Issue by xmedr** [Metro-Records/la-metro-councilmatic#1024]
Go through the following criteria for [WCAG's Robust section](https://www.w3.org/TR/WCAG21/#robust) and improve their issues to meet WCAG requirements:

### Level A
- 4.1.1 Parsing ([used this html validator](https://validator.w3.org/nu/?doc=https%3A%2F%2Flametro.datamade.us%2F)):
  - [ ] PDFs are structured a bit strangely and it affects screen reader performance. Instead of a paragraph being it's own `div`, one word or a whole phrase in the middle of a sentence will be.
  - [x] The calendar pdf also reads things out of order, but it doesn’t look like we have control over that.
  - [x] We’ve got some `img` and other tags that have stray, unneeded ending tags, like the Metro logo `img` tag
  - [x] There are buttons that have `href` attributes. Let's try to change those to just `a` tags
  - [x] Figure out if we need `autocomplete` attributes on radio buttons. If we do then keep them. If not, we can remove.
  - [x] Many `small` tags are encasing `div`s, whereas the `div`s should encase the `small`s
  - [x] Some table elements are missing a `scope` attribute
  - [x] The board report search bar has an `aria-describedby` that doesn’t lead to anything. This is identical to the issue for 3.3.2 Labels or Instructions
  - [x] Run the validator again to see what else can use some improvement.

- 4.1.2 Name, Role, Value:
  - [x] Consider replacing the `title` attribute on various elements with a `tooltip` instead. [Suggestion taken from this site](https://inclusive-components.design/tooltips-toggletips/).
  - [x] Many `div`s may be able to be changed into more semantic tags like sections, articles, etc.
  - [x] ~~The buttons above the board members listing page map are `label`s instead of `button`s. Consider making changing that to be more semantic.~~ These labels contain inputs, so this is semantically correct.

### Level AA
- 4.1.3 Status Messages:
  - [x] Can add an [aria role of alert](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA19) to errors

Connects #1012

**Comment by xmedr** [comment]
Closed by #1088
